### DISCUSSION TITLE
Topic Confused

### DISCUSSION THREAD

**Discussion by loki-qi-fe** [WalletConnect/walletconnect-monorepo#3475] [Developer Support]
- check Ethereum on react-app.walletconnect.com. 

- scan&approve, return the topic **2a4e1afe0b302cde0d2fac3d7a1f70ed4a288cfb21b5746fb83e6a54538889fd**. 

- Immediately，click personal_sign on react-app.walletconnect.com

<img width="522" alt="image" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/131326858/877681f4-3083-4354-a813-ce12c0e7edac">

- check the log in console, session confused.  topic:**5c8050e0e3b63421b783342f7a14f6dd76d995c9a9af2e0e7b7fef0bb6930cff**

<img width="1821" alt="image" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/131326858/2ae85d93-83fb-42ec-83c4-9b1e6dde34fd">

the topic is not matched.

sos~

**Comment by loki-qi-fe** [comment]
I also checked the site, react-app.walletconnect.com

when SessionSettle:
the site generate No.13 message as below,
the android client logged onSeesionSettleResponse with topic **13129ead35a8bcc48ac64f692ccc99bcb0898b078ee4d02a4d7891d0b1c9bcb6**

then Click **personal_sign** as SessionRequest:
the site generate No.14 message as below with topic **13129ead35a8bcc48ac64f692ccc99bcb0898b078ee4d02a4d7891d0b1c9bcb6**
the android client logged onSessionRequest with the unexpected topic **0f252facb737e00d248fa6665a4e7d26b272ebbfec073d0e26dbe89b1cff62ae**, why not ....**bcb6**???


<img width="1860" alt="image" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/131326858/3bcb731c-1915-4f6a-bfad-6cda687cbe37">



<img width="1496" alt="image" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/131326858/221c3aae-7bd5-4cf3-b4e2-6d2d8a71ca49">

**Comment by yangkai69** [comment]
I also encountered the same problem, please help.
My error is as follows：
 UncaughtException:com.walletconnect.android.internal.common.exception.CannotFindSequenceForTopic: Cannot find sequence for given topic:
